sdk/ai/ai-inference-rest/review/ai-inference.api.md

```diff
@@ -241,7 +220,7 @@ export interface FunctionCallOutput {
 export interface FunctionDefinition {
     description?: string;
     name: string;
-    parameters?: unknown;
+    parameters?: Record<string, unknown>;
```

Comment:
This is a change I'm likely to revert in TypeSpec (it was just an experiment). Let me get back to you on this. Please don't complete the PR before we sort this out. Thanks!